client-gateway/bot/passenger.py

```
@@ -283,74 +193,15 @@ async def handle_order_status(update: Update, context: ContextTypes.DEFAULT_TYPE
-                order = user_orders.get(chat_id) or {}
-                # Log request before submitting to TripService
->>>>>>> main
                 logger.info(
                     "Trip request confirmed: chat_id=%s pickup=%s dropoff=%s comment=%s",
                     chat_id, order.get('pickup'), order.get('dropoff'), order.get('comment')
                 )
 
-<<<<<<< HEAD
+                start_time = time.time()
                 result = await submit_trip_request(chat_id, order)
+                latency = int((time.time() - start_time) * 1000)  # в мілісекундах
```

Flake8 (7.3.0)

[error] local variable 'latency' is assigned to but never used

(F841)